### ISSUE TITLE
http://changelog.com/rss stopped working

### ISSUE THREAD

**Issue by gpakosz** [thechangelog/changelog.com#369]
I subscribed to http://changelog.com/rss a while ago but the feed stopped working after April, 30th.

Can you please make the URL work again? Likely through 2 `302` redirects: `http` → `https` then `/rss` → `/feed`.

Thanks in advance!

**Comment by jerodsanto** [comment]
Hmm any idea why both of these redirects would be broken @gerhard? 

- `curl` to `http://changelog.com` returns a `200` but should be a `301` to `https://changelog.com`
- `curl` to `https://changelog.com/rss` returns a `404` but should be a `302` to `https://changelog.com/feed`

The latter is handled by `ChangelogWeb.Plug.Redirects`, not sure where the former is handled in the new setup...

**Comment by gerhard** [comment]
I said **maybe**. Will know for sure in a few minutes.

**Comment by gerhard** [comment]
The http -> https redirect is a bit trickier @jerodsanto, because we now have the CDN in front (this redirect needs to happen in the CDN).

Last time I tried this, I created an infinite redirect loop, even though I'm sure that I read the documentation correctly. I would like to play a bit more with the CDN config, in a new service (not prod), and when I have the http -> https redirect nailed down, I will apply it to prod too.

For now, we are re-building the availability SLI, so the royal we are more risk-averse for the next three (3) months 😉
